grammars/jsdoc.cson

```diff
@@ -252,28 +275,29 @@
         'name': 'variable.other.jsdoc'
       }
       {
-        'name': 'variable.other.jsdoc'
         'begin': '\\['
-        'end': '\\]|(?=\\*/)'
+        'beginCaptures':
+          '0':
+            'name': 'variable.other.jsdoc'
+        'contentName': 'variable.other.jsdoc'
```

Comment:
It's to keep `invalid.illegal.syntax.jsdoc` from being nested inside `variable.other.jsdoc`. For themes that prioritise the former over the latter, we don't want invalid text being highlighted as valid.